src/server/server.js

```diff
@@ -68,8 +68,8 @@ app.get('/football/teams/:teamid/players', (req, res) => {
   searchFootballTeamPlayers(res, req.params.teamid);
 });
 
-app.get('/journalism/teams/:teamname', (req, res) => {
-  journalismTeam(res, req.params.teamname);
+app.get('/journalism/teams/:teamname/:footballDataOrgTeamId', (req, res) => {
+  journalismTeam(res, req.params.teamname, req.params.footballDataOrgTeamId);
```

Comment:
This route rubs me the wrong way, that you'd pass in both a teamname and an id. Still, it's the easier way to do it so leave it as is